api_spec.yaml

```diff
@@ -1832,6 +1752,89 @@ paths:
                                 type: string
                                 enum:
                                     - Unauthorised access
+    /dashboard/api/tenants/core-config/list:
+        get:
+            tags:
+                - Multitenancy
+            summary: Gets all the possible core config options.
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    oneOf:
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - string
+                                                            - boolean
+                                                            - number
+                                                        description: Data type of the property
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - enum
+                                                        description: Data type of the property
+                                                    options:
+                                                        type: array
+                                                        items:
+                                                            type: string
```

Comment:
could options be of other types ? verify once

Reply:
Yes I checked there are only string type properties for enum here:

https://github.com/supertokens/supertokens-core/blob/master/config.yaml